horace_core/sqw/@sqw/symmetrise_sqw.m

```diff
@@ -237,10 +241,10 @@
                        'Expected: %d, received: %d', fold, numel(sym)])
             end
 
-            for i = 1:fold-1
+            for i = 3:fold
                 % If not regular fold
-                if sym{i+1}.theta_deg ~= sym{2}.theta_deg || ...
-                        sym{i+1}.theta_deg / fold ~= sym{2}.theta_deg
+                if sym{i}.theta_deg ~= sym{2}.theta_deg && ...
+                        sym{i}.theta_deg / (i-1) ~= sym{2}.theta_deg
```

Comment:
It appears there are 2 legal cases - either all the syms have the same theta OR they increase uniformly, x, 2x, 3x, 4x etc. This condition is an AND - which may be ok as neither condition is met, but not so readable for maintenance. 

Also the use of integer division with reals possibly rounded makes me nervous.

Also it is unclear what happens to sym{1} in either of its cases.

At least more explanation please, also check for errors.